docs/api/remix.md

```diff
@@ -2236,6 +2236,140 @@ return json(data, {
 });
 ```
 
+### `<Outlet context />`
+
+This component is a wrapper around React Router's Outlet with the ability to pass UI state down to nested routes.
+
+<docs-warning>You can use this for loader data, but you don't need to. It's easier to access all loader data in any component via [`useLoaderData`](#useloaderdata) or [`useMatches`](#usematches).</docs-warning>
+
+Here's a practical example of when you may want to use this feature. Let's say you've got a list of companies that have invoices and you want to display those companies in an accordion. We'll render our outlet in that accordion, but we want the invoice sorting to be controlled by the parent (so changing companies preserves the invoice sorting). This is a perfect use case for `<Outlet context>`.
+
+```tsx filename=app/routes/companies.tsx lines=[6,27-30,35-41,50-54,65]
+import type { LoaderData } from "remix";
+import {
+  json,
+  useLoaderData,
+  useParams,
+  Outlet
+} from "remix";
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel
+} from "@reach/accordion";
+import { getCompanies, Companies } from "~/utils/companies";
+
+type LoaderData = {
+  companies: Array<Companies>;
```

Comment:
As this is a simple type, you can just do

```suggestion
  companies: Companies[];
```

Reply:
Nope. [I don't like that syntax](https://www.briefs.fm/3-minutes-with-kent/222) 😅

Reply:
glad i'm not alone in that 😆

Reply:
You two are `weirdo[]`. I love that syntax.